Voting - refactoring step #1

korczis opened the pull request.

botwillacceptanything commented:
#### :ballot_box_with_check: Voting procedure reminder:
To cast a vote, post a comment containing `:+1:` (:+1:), or `:-1:` (:-1:).
Remember, you **must :star:star this repo for your vote to count.**

All comments within this discussion are searched for votes, regardless of the time of posting.
You can cast as many votes as you want, but only the last one will be counted.
(You may consider editing your comment instead of adding a new one.)
Comments containing both up- and down-votes are disregarded.
PR authors automatically count as a :+1: vote.

A decision will be made after this PR has been open for **15** minutes (plus/minus **10** percent, to avoid people timing their votes), and at least **8** votes have been made.
A supermajority of **65%** is required for the vote to pass.

*NOTE: the PR will be closed if any new commits are added after:* 2411e8a2f51f5b07d3437ddd485b4640db0fb496

SpenserJ commented:
Did any code change in `main.js` or `voting.js`, or was it just moving the file to the new location and wrapping it in AMD?

korczis commented:
I just moved the folder, used AMD so instead of config.js is config/index.js loaded and fixed the paths in config/index.js, prepared section in config.sample.js for voting constants

botwillacceptanything commented:
#### :warning: This PR has been modified and is now being closed.

To prevent people from sneaking in changes after votes have been made, pull requests can't be committed on after they have been opened. Feel free to open a new PR for the proposed changes to start another round of voting.
